docs/new-idp-requirements.md

```diff
@@ -0,0 +1,40 @@
+# New IDP Requirements
+
+## Summary
+
+This document describes the minimum requirements for adding a new IDP (Identity Provider) to the Sigstore Public Good Deployment.
+
+Adding a new IDP option to Fulcio helps drive adoption of signing and verification for software artifacts using Sigstore Public Good. Because identity is a critical component of the system, it's important that new IDPs meet the minimum set of requirements to ensure the security and reliability of the system.
+
+You should also reference the [Fulcio - ODIC.md](https://github.com/sigstore/fulcio/blob/main/docs/oidc.md) documentation for additional requirements for the type of IDP you're looking to integrate. The current two likely types of IDPs are:
+
+- `Email`    - Email-based OIDC providers use the user’s email as the subject of the certificate.
```

Comment:
Can we change this to also mention service accounts? It can either be user-based authentication or machine identity for SAs

Reply:
I think I captured what you're looking for but let me know if not